test/units/testsuite-75.sh

```diff
@@ -162,10 +162,12 @@ ip link del hoge.foo
 ### SETUP ###
 # Configure network
 hostnamectl hostname ns1.unsigned.test
-{
-    echo "10.0.0.1               ns1.unsigned.test"
-    echo "fd00:dead:beef:cafe::1 ns1.unsigned.test"
-} >>/etc/hosts
+cat >>/etc/hosts <<'EOF'
```

Comment:
EOF in quotes has special meaning — variable substitution is suppressed. You're not making use of this here, so please drop the quotes.

Reply:
I strongly prefer and recommend to use quoted EOF **unless** lines of the here-document should undergo parameter expansion, command substitution, and arithmetic expansion.

Reply:
Dunno, that'd be very verbose. But the code should be consistent. So either we do this everywhere, or nowhere. There's a bunch of `<<EOF` in this file, all without quotes.